src/main/resources/assets/crimson/lang/en_us.json

```diff
@@ -22,6 +22,9 @@
   "block.crimson.vanadinite_ore": "Vanadinite Ore",
   "block.crimson.crimson_thorn": "Crimson Thorn",
   "block.crimson.crimson_log": "Crimson Log",
+	"block.crimson.stripped_crimson_log": "Stripped Crimson Log",
+  "block.crimson.crimson_wood": "Crimson Wood",
+	"block.crimson.stripped_crimson_wood": "Stripped Crimson Wood",
```

Comment:
wtf is this indentation

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.